src/main/java/com/gd/clinic/security/entity/Role.java

```diff
@@ -0,0 +1,27 @@
+package com.gd.clinic.security.entity;
+
+import com.gd.clinic.security.enums.RoleName;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Data
+@Entity
+public class Role {
```

Comment:
actually no reasons to have the role as a separate table in database. you can just create `Role` enum (just rename current `RoleName`) and use it in your entities directly